### PR TITLE
[AOP-2820] Source and destination info

### PR DESCRIPTION
**Pull request by paigehargrave**

### CODE REVIEW

File: modules/ROOT/pages/prereq-network.adoc [hunk 1]
```diff
@@ -17,31 +17,51 @@ restarts. If IP addresses are not persistent between reboots, the cluster can fa
...
-|TCP | 30000-32767 | Internal services port range
-|TCP | 61008-61010 | Installer port ranges (only used during install)
-|TCP | 61022-61024 | Installer port ranges (only used during install)
-|UDP | 8472 | Overlay VXLAN network
-|TCP | 32009 | Ops Center Admin UI
+|Protocol |Port/Range |Purpose |Source |Destination
+|TCP/UDP | 53 | Internal cluster DNS | localhost | localhost
+|TCP | 2379, 2380, 4001, 7001 | etcd distributed database | all | controllers
+|TCP | 4242 | Installer | all | all
+|TCP | 6060 | Health check | all | all
```
Comment (alejandrorascovan) [inline modules/ROOT/pages/prereq-network.adoc]: Remove this port. Not needed

File: modules/ROOT/pages/prereq-network.adoc [hunk 2]
```diff
@@ -17,31 +17,51 @@ restarts. If IP addresses are not persistent between reboots, the cluster can fa
...
+|TCP | 6443 | Kubernetes API server | all | controllers
+|TCP/UDP | 7496 | Serf RPC agent | all | all
+|TCP/UDP | 7373 | Serf RPC agent | localhost | localhost
+|TCP | 10248, 10249, 10250, 10255 | Kubernetes Kubelet | all | all
```
Comment (alejandrorascovan) [inline modules/ROOT/pages/prereq-network.adoc:41]: Add a new line below:
|TCP | 10251, 10252 | Kubernetes controller-manager and scheduler | all | all

File: modules/ROOT/pages/prereq-network.adoc [hunk 3]
```diff
@@ -17,31 +17,51 @@ restarts. If IP addresses are not persistent between reboots, the cluster can fa
...
+|TCP | 5000 | Docker registry | all | controllers
+|TCP | 3009-3011 | Cluster control plane | all | controllers
```
Comment (alejandrorascovan) [inline modules/ROOT/pages/prereq-network.adoc]: 3009-3012

File: modules/ROOT/pages/prereq-network.adoc [hunk 4]
```diff
@@ -17,31 +17,51 @@ restarts. If IP addresses are not persistent between reboots, the cluster can fa
...
-|UDP | 8472 | Overlay VXLAN network
-|TCP | 32009 | Ops Center Admin UI
+|Protocol |Port/Range |Purpose |Source |Destination
+|TCP/UDP | 53 | Internal cluster DNS | localhost | localhost
+|TCP | 2379, 2380, 4001, 7001 | etcd distributed database | all | controllers
+|TCP | 4242 | Installer | all | all
+|TCP | 6060 | Health check | all | all
+|TCP | 6443 | Kubernetes API server | all | controllers
+|TCP/UDP | 7496 | Serf RPC agent | all | all
+|TCP/UDP | 7373 | Serf RPC agent | localhost | localhost
```
Comment (alejandrorascovan) [inline modules/ROOT/pages/prereq-network.adoc]: Remove this port. Not needed

File: modules/ROOT/pages/prereq-network.adoc [hunk 5]
```diff
@@ -17,31 +17,51 @@ restarts. If IP addresses are not persistent between reboots, the cluster can fa
...
+|TCP | 10248, 10249, 10250, 10255 | Kubernetes Kubelet | all | all
+|TCP | 5000 | Docker registry | all | controllers
+|TCP | 3009-3011 | Cluster control plane | all | controllers
+|TCP | 3012 | Cluster control plane | all | all
+|TCP | 3022 | Cluster control plane | all | all
+|TCP | 3023-3026 | Cluster control plane | all | controllers
+|TCP | 3080 | Cluster control plane | external | controllers
+|TCP | 7575 | Cluster control plane | all | all
+|TCP | 7001 | Cassandra | data nodes | data nodes
+|TCP | 7199 | Cassandra | localhost | localhost
```
Comment (alejandrorascovan) [inline modules/ROOT/pages/prereq-network.adoc]: Remove this port. Not needed

File: modules/ROOT/pages/prereq-network.adoc [hunk 6]
```diff
@@ -17,31 +17,51 @@ restarts. If IP addresses are not persistent between reboots, the cluster can fa
...
+|TCP | 9042 | Cassandra | all | all
+|TCP | 18080 | Object store cluster | localhost | localhost
```
Comment (alejandrorascovan) [inline modules/ROOT/pages/prereq-network.adoc]: Remove this port. Not needed.

**Review by alejandrorascovan — DISMISSED** [review]

File: modules/ROOT/pages/prereq-network.adoc (hunk 1 above)
Comment (paigehargrave) [reply modules/ROOT/pages/prereq-network.adoc]: Removed.

File: modules/ROOT/pages/prereq-network.adoc (hunk 3 above)
Comment (paigehargrave) [reply modules/ROOT/pages/prereq-network.adoc]: Please see the following line.

File: modules/ROOT/pages/prereq-network.adoc (hunk 4 above)
Comment (paigehargrave) [reply modules/ROOT/pages/prereq-network.adoc]: Removed.

File: modules/ROOT/pages/prereq-network.adoc (hunk 2 above)
Comment (paigehargrave) [reply modules/ROOT/pages/prereq-network.adoc:41]: Added.

File: modules/ROOT/pages/prereq-network.adoc (hunk 5 above)
Comment (paigehargrave) [reply modules/ROOT/pages/prereq-network.adoc]: Removed.

File: modules/ROOT/pages/prereq-network.adoc (hunk 6 above)
Comment (paigehargrave) [reply modules/ROOT/pages/prereq-network.adoc]: Removed.

File: modules/ROOT/pages/prereq-network.adoc (hunk 3 above)
Comment (alejandrorascovan) [reply modules/ROOT/pages/prereq-network.adoc]: Let's consolidate them

File: modules/ROOT/pages/prereq-network.adoc [hunk 7]
```diff
@@ -17,31 +17,47 @@ restarts. If IP addresses are not persistent between reboots, the cluster can fa
...
 The version of Kubernetes supported by Anypoint Platform PCE uses an overlay VXLAN and UDP 
-transport to encapsulate traffic. There is direct communication between components of the cluster. The following table 
-lists the ports used for inter-host communication. You must ensure that all of the following ports are 
+transport to encapsulate traffic. There is direct communication between cluster components. 
+
+The following table 
+lists the ports used for inter-host communication with the following source and destination descriptions:
+
+all:: Any node that is a member of the cluster.
+external:: Any source outside the cluster
```
Comment (wwarner-salesforce) [inline modules/ROOT/pages/prereq-network.adoc]:
```suggestion
external:: Any source outside the cluster.
```

File: modules/ROOT/pages/prereq-network.adoc [hunk 8]
```diff
@@ -17,31 +17,47 @@ restarts. If IP addresses are not persistent between reboots, the cluster can fa
...
+localhost:: The port is used only within the host where the request started.
+controllers:: Nodes that are designated with the controller or master role
```
Comment (wwarner-salesforce) [inline modules/ROOT/pages/prereq-network.adoc]:
```suggestion
controllers:: Nodes that are designated with the controller or master role.
```

**Review by wwarner-salesforce — APPROVED** [review]

File: modules/ROOT/pages/prereq-network.adoc (hunk 3 above)
Comment (paigehargrave) [reply modules/ROOT/pages/prereq-network.adoc]: Done!

**Pull request merged**